core/src/syn/v1/part/mod.rs

```diff
@@ -67,10 +67,20 @@ pub fn changefeed(i: &str) -> IResult<&str, ChangeFeed> {
 	let (i, _) = tag_no_case("CHANGEFEED")(i)?;
 	let (i, _) = shouldbespace(i)?;
 	let (i, v) = cut(duration)(i)?;
+
+	let (i, opt) = opt(|i| {
+		let (i, _) = shouldbespace(i)?;
+		let (i, _) = tag_no_case("INCLUDE")?;
```

Comment:
Suggesting fixes for my bad suggestions:
```suggestion
		let (i, _) = tag_no_case("INCLUDE")(i)?;
```

Reply:
Ah I didnt realise you commented this 😂 I was digging around trying to figure it out. Got there in the end. Thank you!